Insta-Bot-Follow-SendMsg/instabot.py

```diff
@@ -0,0 +1,54 @@
+from selenium import webdriver
+import time
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from openpyxl import load_workbook
+from selenium.webdriver.common.action_chains import ActionChains
+
+browser = webdriver.Chrome(ChromeDriverManager().install())
+time.sleep(10)
+users=[]
+users.append(input("Enter Users Username Whom You Want to Follow and Send Msg "))
```

Comment:
how this will take multiple users? It will take all the comma-separated users as strings.

Reply:
@kaustubhgupta updated